src/js/hooks.ts

```diff
@@ -5,3 +5,22 @@ import type { RootState, AppDispatch } from './store';
 // refer to https://react-redux.js.org/using-react-redux/usage-with-typescript#define-typed-hooks for more info
 export const useAppDispatch: () => AppDispatch = useDispatch;
 export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector;
+
+// ################### TRANSLATION HOOKS ###################
+import { useTranslation } from 'react-i18next';
+import { DEFAULT_TRANSLATION, NON_DEFAULT_TRANSLATION } from '@/constants/configConstants';
+import { NamespaceTranslationFunction } from '@/types/translation';
```

Comment:
Place imports at the top

Reply:
it is to separate both, in future when we get more hooks they would end up going into a new folder and separate files